File: data/src/main/java/com/mbmc/template/data/api/ApiService.java
package com.mbmc.template.data.api;

import com.mbmc.template.data.entity.RepoData;
import io.reactivex.Observable;
import retrofit2.http.GET;
import retrofit2.http.Path;

import java.util.List;

public interface ApiService {

    @GET("users/{handle}/repos")
    Observable<List<RepoData>> getRepos(@Path("handle") String handle);

}

File: data/src/main/kotlin/com/mbmc/template/data/api/ApiService.kt
package com.mbmc.template.data.api

import com.mbmc.template.data.entity.RepoData
import io.reactivex.Single
import retrofit2.http.GET
import retrofit2.http.Path

interface ApiService {
    @GET("users/{handle}/repos")
    fun getRepos(@Path("handle") handle: String): Single<List<RepoData>>
}
